formats/gltf/extensions.go

```diff
@@ -3,10 +3,18 @@ package gltf
 import "image/color"
 
 type MaterialExtension interface {
-	ExtensionID() string
+	MaterialExtensionID() string
```

Comment:
I think I'd prefer to keep the `ExtensionID` signature. I could imagine a later refactor where we introduce a general `Extension` interface, that would allow us to do something like, iterate over all extensions and build a list of used/required.

To keep Material and Texture extensions separate, we could change the Material interface's `ToExtensionData` to something like `MaterialExtensionData`